### PR TITLE
fix(button): give the button default theme,reset missing scripts

### PR DESCRIPTION
**Pull request by yzh990918**
I don't know why the previously committed code was removed.

### CODE REVIEW

File: packages/components/button/src/index.vue [hunk 1]
```diff
@@ -10,6 +10,7 @@ const binds = Object.assign({}, useAttrs(), props.to ? { href: props.to } : {})
 
 <template>
   <component
+    :o="o"
```
Comment (zyyv) [inline packages/components/button/src/index.vue]: I dislike `o` property on component, i change it to `useAttrs`.
`const binds = Object.assign({}, useAttrs(), props.to ? { href: props.to } : {})`

How do you think? `o` property are too abstract, i think it works as a unocss variant, not a componet property. 
The same property has `border` ...

**Comment by priority3** [comment]
It seems that there are some build errors , it's strage too for me🤣, when I just debug, I found that the type prompt did not work

**Comment by yzh990918** [comment]
> It seems that there are some build errors , it's strage too for me🤣, when I just debug, I found that the type prompt did not work

Maybe some dependencies improved caught it.

File: packages/components/package.json [hunk 2]
```diff
@@ -32,7 +32,7 @@
   ],
   "sideEffects": false,
   "scripts": {
-    "build": "vite build && pnpm run --filter \"../onu-ui\" build",
+    "build": "vite build && vue-tsc --declaration --emitDeclarationOnly --skipLibCheck && pnpm run --filter \"../onu-ui\" build",
```
Comment (zyyv) [inline packages/components/package.json]: I'm not sure what `vue-tsc` does, do we really need it?

File: packages/components/button/src/index.vue (hunk 1 above)
Comment (yzh990918) [reply packages/components/button/src/index.vue]: But I think button should have its default style when I use `<o-button>test</o-button>`.

File: packages/components/package.json (hunk 2 above)
Comment (yzh990918) [reply packages/components/package.json]: This is a solution to solve the problem that normal component type declaration files cannot be generated using `vite-plugin-dts`. Using `vite-plugin-dts`  causes the props type of some components to change to any.  But this is really a temporary plan, I will modify it.

File: packages/components/button/src/index.vue (hunk 1 above)
Comment (priority3) [reply packages/components/button/src/index.vue]: > But I think button should have its default style when I use `<o-button>test</o-button>`.

const binds = Object.assign({o: 'primary' }, useAttrs(), props.to ? { href: props.to } : {}) 🤣

Comment (zyyv) [reply packages/components/button/src/index.vue]: I remember which component I wrote on before, but I didn't find it. 😅

```diff
<component
-    :o="o"
+   o-primary
/>
```
How about this? If there is an attribute behind it, it should be overwritten.

**Comment by yzh990918** [comment]
https://github.com/qmhc/vite-plugin-dts/issues/129 @chris-zhu  `vue-tsc` is much easier to use and the declaration file is more accurate.

**Pull request closed without merging**